gapic/api/artman_clouddebugger.yaml

```diff
@@ -0,0 +1,30 @@
+common:
+  api_name: google-clouddebugger-v2
+  proto_gen_pkg_deps:
+    - google-common-protos
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2
+  service_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/clouddebugger.yaml
+  gapic_api_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml
+  output_dir: ${REPOROOT}/artman/output
+  enable_batch_generation: True
+java:
+  final_repo_dir: ${REPOROOT}/google-cloud-java/google-cloud-clouddebugger
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-clouddebugger
+go:
+  final_repo_dir: ${REPOROOT}/gapi-clouddebugger-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-clouddebugger
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-clouddebugger
+ruby:
+  final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-clouddebugger
+  skip_packman: True
```

Comment:
I don't think this is problematic.

- There is a small, finite number of existing APIs that have handwritten client libraries, for which this flag needs to be set by hand.
- The cost of creating new handwritten client libraries is very high compared to the cost of manually setting this flag.
- There will always be some configuration required for each API we generate.
- If there are values that are always deterministically knowable from the input, they should be set in the code rather than through configuration.

(That's not to say, of course, that we shouldn't make an effort to minimize the amount of configuration work necessary.)